src/lib/shared/input.ts

```diff
@@ -308,23 +352,23 @@ export const inputPhrases: InputPhrases = {
   "input.status.successfulValue": "Successful",
   "input.statusLabel": "Status",
   "input.stepLabel": "Step",
+  "input.timeMeasurement.hoursValue": "Hours",
```

Comment:
should probably just make this `Hours` instead of hoursValue (and elsewhere) since these are used in an array we dont access them directly in the client code

Reply:
@jasoncomes